src/Nexus.UI/Charts/Chart.razor.cs

```diff
@@ -28,6 +28,8 @@ public partial class Chart : IDisposable
     private DateTime _zoomedBegin;
     private DateTime _zoomedEnd;
 
+    private readonly SKRect _defaultZoomBox = new SKRect(0, 0, 1, 1);
```

Comment:
I think this variable should be `private static readonly`. Apart from that, everything looks fine :-)

Reply:
Sorry, wrong account